uploadservice/src/main/java/net/gotev/uploadservice/UploadTaskParameters.java

```diff
@@ -73,6 +74,10 @@ public void addFile(UploadFile file)
         files.add(file);
     }
 
+    public void removeFile(UploadFile file) {
+        files.remove(file);
```

Comment:
For the remove to work without problems, the `UploadFile` class should implement `equals` and `hashCode`. I will take care of this after the merge.